nix/nixpkgs-src.json

```diff
@@ -1,7 +1,7 @@
 {
   "url": "https://github.com/nixos/nixpkgs",
-  "rev": "5d4dbdba68ab42468e65c5a9a9dcd4d6999b0363",
```

Comment:
Why did you have to change this (and `nix/haskell.nix-src.json`)?

Reply:
Don't know, I didn't change it by hand and I'm puzzled as to how nix settings for this repo work.